tests/test_src/test_filehandler/test_filehandler.cpp

```diff
@@ -94,3 +95,71 @@ TEST_P(PemFileReaderParametrized, getSeed_getPublicKey_getBech32Address)
     EXPECT_EQ(pemPubKey, util::hexToBytes(currParam.publicKey));
     EXPECT_EQ(pemBech32Address, currParam.bech32Address);
 }
+
+class KeyFileReaderConstructorFixture : public ::testing::Test
```

Comment:
Never thought that CPP unit tests can be written in such a clean and readable way :) :+1:

Reply:
Thanks a lot, appreciate it !